samples/SampleApp/Startup.cs

```diff
@@ -33,24 +34,33 @@ public void Configure(IApplicationBuilder app, ILoggerFactory loggerFactory)
 
         public static void Main(string[] args)
         {
-            var host = new WebHostBuilder()
-                .UseKestrel(options =>
+            var hostBuilder = new WebHostBuilder().UseKestrel(options =>
+            {
+                options.Listen(IPAddress.Loopback, 5000, listenOptions =>
```

Comment:
For a realistic sample the IP and port need to come from command line / env / config. We need to make that possible in one or two lines. Even if it means having a Listen overload that takes a string for `IP:port` that understands IPv4, IPv6, and *.
@shirhatti 
`options.Listen(config["IPAndPort"], listenOptions => ...`

Note IPEndpoint does not implement Parse so you can't just punt this over there.